std_msgs/Header header

uint32 id
uint32 original_id
string label
float32 score
geometry_msgs/Pose pose

float32 shift
bool visible
time detected_time
